components/HomePage/index.tsx

```diff
@@ -1,10 +1,197 @@
+                  {project.name}
+                  <UI.ProjectNameIcon />
+                </UI.ProjectName>
+              </UI.ProjectLink>
+            </UI.ProjectContainer>
+          ))}
+        </UI.ProjectsList>
+        <SecondaryLink href={LINKS.projects} target="_blank">Explore all</SecondaryLink>
+      </UI.HomeSectionContent>
+    </UI.HomeSectionContainer>
+  );
+}
+
+const WhyUseUMA: React.FunctionComponent = () => {
+  return (
+    <UI.HomeSectionContainer grayBackground>
+      <UI.HomeSectionContent>
+        <UI.HomeSectionHeadline>Why use uma?</UI.HomeSectionHeadline>
+        <UI.UmaFeatures>
+          {UMA_FEATURES.map((feature, idx) => (
+            <UI.UmaFeatureContainer key={idx}>
+              <UI.UmaFeature>
+                <UI.UmaFeatureImgContainer>
+                  <UI.UmaFeatureImg src={feature.icon}></UI.UmaFeatureImg>
+                </UI.UmaFeatureImgContainer>
+                <UI.OOBoxTextContainer>
+                  <UI.BoxTitle>{feature.title}</UI.BoxTitle>
+                  <UI.BoxDescription>{feature.description}</UI.BoxDescription>
+                </UI.OOBoxTextContainer>
+              </UI.UmaFeature>
+            </UI.UmaFeatureContainer>
+          ))}
+        </UI.UmaFeatures>
+      </UI.HomeSectionContent>
+    </UI.HomeSectionContainer>
+  );
+}
+
+const Products: React.FunctionComponent = () => {
+  return (
+    <UI.HomeSectionContainer>
+      <UI.HomeSectionContent>
+        <UI.HomeSectionHeadline>Products</UI.HomeSectionHeadline>
+        <UI.ProductsContainer>
+          {PRODUCTS.map((product, idx) => (
+            <UI.ProductContainer key={idx}>
+              <UI.Product>
+                <UI.ProductCategoryContainer>
+                  {product.categoryIcon}
+                  {product.categoryName}
+                </UI.ProductCategoryContainer>
+                <UI.ProductName>{product.name}</UI.ProductName>
+                <UI.ProductFeaturesList>
+                  {product.features.map((feature, idx) => (
+                    <UI.ProductFeatureItem key={idx}>{feature}</UI.ProductFeatureItem>
+                  ))}
+                </UI.ProductFeaturesList>
+                <UI.ProductLearnMoreButton href={product.href}>Learn more</UI.ProductLearnMoreButton>
+              </UI.Product>
+            </UI.ProductContainer>
+          ))}
+        </UI.ProductsContainer>
+        <UI.ProductsExploreButton href="/products">Explore all products</UI.ProductsExploreButton>
+      </UI.HomeSectionContent>
+    </UI.HomeSectionContainer>
+  );
+}
+
+const GetStarted: React.FunctionComponent = () => {
+  return (
+    <UI.GetStartedContainer>
+      <UI.GetStartedContent>
+          <UI.GetStartedTitle>{`Are you ready to integrate Web3's most versatile oracle?`}</UI.GetStartedTitle>
+          <UI.GetStartedLink href={LINKS.getStarted}>Get started</UI.GetStartedLink>
+      </UI.GetStartedContent>
+    </UI.GetStartedContainer>
+  );
+}
+
+const OOBoxesData = [{
```

Comment:
I like this approach, could we move it somewhere where it is more accessible to non technical people? Like a JSON or a separate file